desktop.css

```diff
@@ -0,0 +1,928 @@
+        text-align: center;
+        color: #FFFFFF;
+    }
+    #item-navigation {
+        width: 255px;
+        height: 40px;
+        display: flex;
+        flex-direction: row;
+        gap: 8px;
+        position: absolute;
+        right: 13%;
+        top: calc(50% - 40px/2);
+    }
+    .navigation-contact {
+        width: 87px;
+    }
+    .navigation-about {
+        width: 69px;
+    }
+    .navigation-contact {
+        width: 83px;
+    }
+    .item-text {
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 600;
+        font-size: 15px;
+        line-height: 20px;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #FFFFFF;
+    }
+    #mail {
+        display: flex;
+        flex-direction: row;
+        align-items: flex-start;
+        padding: 8px;
+        width: 40px;
+        height: 40px;
+        position: absolute;
+        right: 10%;
+        top: calc(50% - 40px/2);
+    }
+    
+    /* Headline section */
+    #headline {
+        position: relative;
+        height: 739px;
+        width: 39%;
+        display: flex;
+        flex-direction: column;
+        gap: 24px;
+        left: 10%;
+        top: 72px;
+        padding: 120px 0px;
+    }
+    .my-name  {
+        width: 100%;
+        height: 160px;
+        font-family: 'Crete Round';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 72px;
+        line-height: 82px;
+        color: #F55800;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+    }
+    .supporting-text{
+        width: 566px;
+        height: 24px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 500;
+        font-size: 32px;
+        line-height: 44px;
+        display: flex;
+        align-items: center;
+        color: #FFFFFF;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+    }
+    .description {
+        width: 566px;
+        height: 123px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 20px;
+        line-height: 32px;
+        color: #F4F5F7;
+        flex: none;
+        order: 2;
+        flex-grow: 0;
+    }
+    .background-image {
+        display: none;
+    }
+    #desktop-headline-image {
+        position: absolute;
+        height: 100%;
+        width: 43%;
+        top: 72px;
+        right: 0px;
+    }
+    .social-media {
+        display: none;
+    }
+    .desktop-social-media {
+        display: flex;
+        flex-direction: column;
+        gap: 24px;
+        position: absolute;
+        top: 300px;
+        left: 3%;
+    }
+    /* Work section */
+    #recent-works {
+        width: 100%;
+        height: 1705px;
+        background-color: #fff;
+        padding: 0px 10%;
+        position: relative;
+        bottom: 80px;
+    }
+    #recent-work {
+        width: 100%;
+        height: 100%;
+        padding: 130px 2%;
+    }
```

Comment:
- [ ] Kindly check the font-size and text-alignment of the recent works title, there are some slight differences between yours and the figma design. 
- [ ] Kindly check and fix the width and height of the recent work image compared to the text on the right side. The height of the text description is suppose to be shorter than the image but is the opposite in your design as shown below :point_down: 


### Your design 

![Screenshot from 2023-01-27 17-56-55](https://user-images.githubusercontent.com/65233860/215164424-151ab821-143d-48d6-aeae-279f5d0f446a.png)

### The figma design 


![Screenshot from 2023-01-27 18-13-37](https://user-images.githubusercontent.com/65233860/215164482-f6b6864f-1d7a-406c-b9e1-c98c1b72c3e4.png)